index.html

```diff
@@ -112,10 +112,11 @@
           </h4>
           <p>
             When the <dfn>share</dfn> method is called with argument
-            <var>data</var>, run the following steps:
+            <var>data</var>, the user agent MUST run the following steps:
```

Comment:
FYI at least in specs that depend on Infra, this is implied: https://infra.spec.whatwg.org/#algorithms